### ISSUE TITLE
introducer.sparecoin.org is not known in DNS

### ISSUE THREAD

**Issue by DaCHRIS** [Spare-Network/spare-blockchain#8]
introducer.sparecoin.org as included in the config.yaml is not even known by the public dns servers

**Comment by almando** [comment]
how to build a introducer? is it possible? i guess if it is really decentralized it would be possible.

**Comment by DaCHRIS** [comment]
> how to build a introducer? is it possible? i guess if it is really decentralized it would be possible.

sure, just start an introducer, it is ready built in 

The problem with this incident is, that the initial introducer is standig in the config.yaml and should be available, your private one is not known...

**Comment by jneuhauser** [comment]
But dns-introducer.sparecoin.org is also listed in the config... Probably this one is sufficient.

```
$ nslookup dns-introducer.sparecoin.org
Server:		127.0.0.53
Address:	127.0.0.53#53

Non-authoritative answer:
Name:	dns-introducer.sparecoin.org
Address: 34.216.131.71
Name:	dns-introducer.sparecoin.org
Address: 34.217.85.127
Name:	dns-introducer.sparecoin.org
Address: 34.219.26.212
Name:	dns-introducer.sparecoin.org
Address: 54.184.188.218
Name:	dns-introducer.sparecoin.org
Address: 54.185.14.61
Name:	dns-introducer.sparecoin.org
Address: 54.190.51.13
Name:	dns-introducer.sparecoin.org
Address: 54.213.57.135
Name:	dns-introducer.sparecoin.org
Address: 18.236.99.235
Name:	dns-introducer.sparecoin.org
Address: 18.237.57.6
```

**Comment by root-spare** [comment]
@jneuhauser, you're absolutely right, we use DNS introducer just for the initial bootstrap of the system, and later peers are discovered via peer-to-peer communication

**Comment by beamercola** [comment]
I'm just getting this

```
nslookup introducer.spare.farm 
Server:		192.168.86.1
Address:	192.168.86.1#53

** server can't find introducer.spare.farm: NXDOMAIN
```

**Comment by horizon6969** [comment]
what is the remedy for it ? 
Farming status: Not synced or not connected to peers
full_node spare.full_node.full_node: ERROR    Exception while querying DNS server: The DNS query name does not exist: dns-introducer.sparecoin.org.

**Comment by horizon6969** [comment]
Looks better:

host dns-introducer.sparecoin.org
dns-introducer.sparecoin.org has address 54.213.57.135
dns-introducer.sparecoin.org has address 34.217.85.127
dns-introducer.sparecoin.org has address 54.184.188.218
dns-introducer.sparecoin.org has address 34.216.131.71
dns-introducer.sparecoin.org has address 18.237.57.6
dns-introducer.sparecoin.org has address 18.236.99.235
dns-introducer.sparecoin.org has address 54.185.14.61
dns-introducer.sparecoin.org has address 54.190.51.13
dns-introducer.sparecoin.org has address 34.219.26.212


Now different error

spare-blockchain/spare/rpc/wallet_rpc_api.py
tx_records: List[TransactionRecord] = await self.service.wallet_state_manager.tx_store.get_farming_rewards()
AttributeError: 'NoneType' object has no attribute 'tx_store'
